### PR TITLE
Update pipeline.md spelling

### PR DESCRIPTION
**Pull request by jorwalk**
Spelling: `communitation` to `communication`

### CODE REVIEW

**Comment by jorwalk** [comment]
fixes `work-around` from PR: https://github.com/kubeflow/website/pull/2198

File: content/en/docs/aws/pipeline.md
Comment (8bitmp3) [inline content/en/docs/aws/pipeline.md:9]: Good catch @jorwalk - thank you so much!

Can you please change `work-around` to `workaround`? I noticed the maintainers don't use the dash (https://github.com/kubeflow/website/search?q=workaround).

Appreciate your help!

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
Good catch @jorwalk - thank you so much!

Can you please change `work-around` to `workaround`? I noticed the maintainers don't use the dash (https://github.com/kubeflow/website/search?q=workaround).

Appreciate your help!

File: content/en/docs/aws/pipeline.md
Comment (jorwalk) [reply content/en/docs/aws/pipeline.md]: Thank you for the quick review. I have updated `work-around` to `workaround`

**Comment by 8bitmp3** [comment]
/assign @jeffwan

Please take a look and approve, if all is OK here. I think @PatrickXYS sent a PR to amend the OWNERS file to help with the workload.

**Review by 8bitmp3 — APPROVED** [review]
Thank you @jorwalk

**Comment by jorwalk** [comment]
Thank you for allowing me to contribute.

**Comment by jorwalk** [comment]
Resolved merge conflict.

**Pull request merged**
